code/modules/admin/watchlist.dm

```diff
@@ -36,6 +36,8 @@
 	if(browse)
 		watchlist_show(target_sql_ckey)
 
+	add_note(target_ckey, "Added to Watchlist - [reason]", null, usr.ckey, 0, null, 0)
```

Comment:
Daily reminder that not disclosing metagame notes is dumb anyway because

A) It gives players no chance to explain themselves

B) If they are actually metagaming and they know you're on to them they'll either stop (problem solved) or do something stupid and get caught anyway (Snerdly)